### PR TITLE
NO-JIRA: Dependabot#3659

### PR DESCRIPTION
**Pull request by Patryk-Stefanski**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes # [3659](https://github.com/openshift/hypershift/pull/3659)

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: cmd/install/assets/cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsclustertemplates.yaml [hunk 1]
```diff
@@ -440,7 +441,7 @@ spec:
             - template
             type: object
         type: object
-    served: true
+    served: false
```
Comment (jparrill) [inline cmd/install/assets/cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsclustertemplates.yaml:444]: What this changes from the current workflow?

**Review by jparrill — CHANGES_REQUESTED** [review]
Additionally please squash the commits :).

File: cmd/install/assets/cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsclustertemplates.yaml (hunk 1 above)
Comment (Patryk-Stefanski) [reply cmd/install/assets/cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsclustertemplates.yaml:444]: It disables the use of v1beta1 CRD version.

**Pull request merged**